fix panic

```
panic: runtime error: invalid memory address or nil pointer dereference	
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0xf15db0]	
	
goroutine 45 [running]:	
github.com/kilnfi/cosmos-validator-watcher/pkg/rpc.(*Node).syncBlocks(0xc0005c0e10, {0x14403f0, 0xc0005cc2d0})	
	/go/src/app/pkg/rpc/node.go:273 +0xf0	
github.com/kilnfi/cosmos-validator-watcher/pkg/rpc.(*Node).Start(0xc0005c0e10, {0x14403f0, 0xc0005cc2d0})	
	/go/src/app/pkg/rpc/node.go:159 +0x798	
github.com/kilnfi/cosmos-validator-watcher/pkg/rpc.(*Pool).Start.func1()	
	/go/src/app/pkg/rpc/pool.go:35 +0x28	
golang.org/x/sync/errgroup.(*Group).Go.func1()	
	/go/pkg/mod/golang.org/x/sync@v0.3.0/errgroup/errgroup.go:75 +0x64	
created by golang.org/x/sync/errgroup.(*Group).Go	
	/go/pkg/mod/golang.org/x/sync@v0.3.0/errgroup/errgroup.go:72 +0xa5
```

--- a/pkg/rpc/node.go
+++ b/pkg/rpc/node.go
@@ -261,6 +261,10 @@ func (n *Node) syncBlocks(ctx context.Context) {
 	}
 
 	currentBlock := currentBlockResp.Block
+	if currentBlock == nil {
+		log.Error().Err(err).Msgf("no block returned when requesting latest block")
+		return
+	}
 
 	// Check the latest known block height
 	latestBlockHeight := int64(0)

